old_ZipCodesTable.sql

```diff
@@ -142,6 +142,7 @@ INSERT INTO zip_codes (id, sid, default_state, city, default_type, zip_code, har
 (5155, 7, 'CO', 'DENVER', 'STANDARD', '80294', '5b'),
 (5156, 7, 'CO', 'DENVER', 'STANDARD', '80299', '5b')
 (5166, 7, 'CO', 'DURANGO', 'STANDARD', '81301', '6a'),
+-- 5166 was causing an error on terminal

```

Comment:
This was very weird, still don't know why it gave an error 🤔

Reply:
There's no comma at the end of line 143.  It was something we were able to fix when we imported the data into the database.  😄